samples/app-complete-auth/nodejs/index.js

```diff
@@ -14,20 +14,10 @@ const axios = require('axios')
 // Read botFilePath and botFileSecret from .env file.
 const ENV_FILE = path.join(__dirname, '.env');
 require('dotenv').config({ path: ENV_FILE });
-const PORT = process.env.PORT || 3978;
-const server = express();
```

Comment:
Why did you remove this part?

Reply:
it was not removed but, just changes the position 
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/109147706/e09d14d8-3de4-4ab2-b516-2124a0e2140b)